### ISSUE TITLE
[Feature Request] Add music support with Lidarr integration

### ISSUE THREAD

**Issue by jamess60** [sct/overseerr#3819]
### Description

For those unaware, Lidarr is basically Sonarr/Radarr but for music. You can track entire bands/artists and it works pretty much the same way as the TV/Movie counterparts. 

### Desired Behavior

Imagine a third tab on the sidebar titled Music, that functions exactly the same as Series/Movies, and polls Lidarr for the content. 

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by r-netizen** [comment]
-> #3800

**Comment by jamess60** [comment]
Ah very nice! 
So from what I can see, its "developed", but waiting to be merged/tested?

**Comment by Michael5564445** [comment]
> Ah very nice! So from what I can see, its "developed", but waiting to be merged/tested?

https://hub.docker.com/r/ano002/overseerr-lidarr
There is already a container, but it is a TEST VERSION

**Comment by ano0002** [comment]
> Ah very nice! 
> So from what I can see, its "developed", but waiting to be merged/tested?

Yes exactly, I did a final build 2 days ago and am now looking for testers to fix as many bugs as possible to make the merging easier

**Comment by Mordrin90** [comment]
I did a pull on this version to run along side my prod version after updating to dev, I was hoping the merge had already happened.
Did a bunch of testing over the past hour or so, haven't encountered anything to crazy issue wise. Duplicate search results and search results without images. Aside from that works as expected, any timeframe on when you'd be looking at merging with the standard branch?

**Comment by PixNyb** [comment]
Any updates on this? Really eager to start using this

**Comment by albatross1** [comment]
Any update on this? Since June?

**Comment by inform880** [comment]
Also would love this feature! Anything I can do to help move this along?

**Comment by jochman** [comment]
Commenting so it won't be stale.

Thank you guys ❤️

**Comment by brandnkdavis** [comment]
I would absolutely love to see this feature added! If I can help in any way please let me know.
